.circleci/workflows.yml

```diff
@@ -310,7 +312,9 @@ jobs:
                 name: PyTest Integration Test
                 # yamllint disable rule:line-length
                 command: |
-                  PATH="venv/bin:$PATH" script/entrypoint -m 'integration'
+                  PATH="venv/bin:$PATH" script/entrypoint -m 'integration' -n 8 \
```

Comment:
Running these in parallel now is about a minute faster. I'm wondering if previously the UDF parsing was done by thread and was slower because of that.